.github/workflows/pint.yml

```
@@ -0,0 +1,21 @@
+name: Run code formatters
+
+on:
+  workflow_call:
+
+jobs:
+  pint:
+    name: "Pint"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)